srcDoc/custom_types.rst

```diff
@@ -0,0 +1,578 @@
+.. _section-custom-types:
+
+============
+Custom Types
+============
+
+*RTI Perftest* allows the user to add their own `idl` types, instead of using
+the one provided by default. The purpose of this feature is to get a closer
+approximation to the performance behavior of the user's final system. The main
+reason why using one type or another will result in different outputs for latency
+and throughput is the serialization and deserialization times.
```

Comment:
*RTI Perftest* allows the user to test the performance of their own datatypes defined via 'idl', instead of 
the one provided by default. The purpose of this feature is to get a closer
approximation to the performance behavior of the user's final system. The main
reason why using one type or another will result in different results for latency
and throughput is that serialization and deserialization performance depends on the complexity of the datatype.

Reply:
By the way, is the testing of customDatatypes only supported in traditional C++?  Not in C, Modern C++ or Java?  If that is the case, should state that here as well.

Reply:
Done.